.bazelrc

```diff
@@ -3,6 +3,7 @@
 # see https://docs.bazel.build/versions/master/guide.html#bazelrc
 #
 # The full list of Bazel options: https://docs.bazel.build/versions/master/command-line-reference.html
+build --cxxopt="-std=c++17"  --host_cxxopt="-std=c++17" --client_env=BAZEL_CXXOPTS=-std=c++17
```

Comment:
We probably don't have much C++ code in this repo but lets just set c++20 here so we're consistent?

Reply:
Also where did you get `--client_env` from? I'm not familiar with that flag.

I think it's redundant anyway since you specified `--cxxopt` - but I *think* you want `repo_env` here because this environment value is accessed as part of the evaluation of the repository rule that sets up bazel's default c++ toolchain.

Reply:
SG! I didn't want to be too greedy on the first stab. If I don't move to g++-10, it complains about c++2a vs. c++20. I added the g++ line, your call if we should go in a different direction.